Client/Client.Qt/registration.cpp

```diff
@@ -1,37 +1,41 @@
 #include "registration.hpp"
-#include "Utility/UserDataValidation.hpp"
 
 #include <QtEvents>
+#include <iostream>
```

Comment:
Do we need iostream header in this place?

Reply:
There is below used as debug